### DISCUSSION TITLE
115挂载完用infuse搜刮秒拦截

### DISCUSSION THREAD

**Discussion by iumuu** [AlistGo/alist#7423] [Q&A]
刚更新完cookie，115官网及alist能正常打开，一用infuse搜刮视频秒出现:"很抱歉，由于您访问的URL有可能对网站造成安全威胁，您的访问被阻断。",block_url_tips:"当前网址: ",block_time_tips:"请求时间: ",block_traceid_tips:"您的请求ID是: "}，，，试了网页端，ios端，微信小程序端的cookie码，都是一搜刮就出错，

**Comment by pegasus503** [comment]
是的 最近115风控得一B，我刮削小日子的动作片，秒封

**Reply by iumuu** [reply]
感觉调整一下应该可以，但是我不知道怎么搞，cd2就可以刮出来，但是速度没有alist快

**Comment by pegasus503** [comment]
是的 最近115风控得一B，我刮削小日子的动作片，秒封
